prototypes/src/genImage/GenImage.jsx

```diff
@@ -0,0 +1,27 @@
+import styles from "./GenImage.module.css"
+import {useState} from "react"
+
+
+export default function GenImage (props) {
+
+    const [pointerCor, setPointerCor] = useState([20, 0])
+
+    const [notes, setNotes] = useState([])
+
+    return (
+        <div className={styles.outline}>
+
+            <div className={styles.click_indicator} style={{ left: `${pointerCor[0]}px`, top: `${pointerCor[1]}px` }}></div>
+            <img
+
+                onClick={(e) => {
+                    const rect = e.target.getBoundingClientRect()
+
+                    // offset the new location by 10 to account for scaling up of image, pointer is too precise when the image is about 480x480px
+                    setPointerCor([e.clientX - rect.left + 10,  e.clientY - rect.top + 10])
```

Comment:
We want to avoid using "magic numbers" in code. Would recommend creating a `const LOCATION_OFFSET = 10` and using that var instead